### ISSUE TITLE
lomap build is failing

### ISSUE THREAD

**Issue by jchodera** [omnia-md/conda-recipes#755]
@davidlmobley : The `lomap` build is [failing](https://travis-ci.org/omnia-md/conda-recipes/jobs/219832950#L3897-L3924):
```
===== testing package: lomap-0.0.0-py27_1 =====
import: 'lomap'
/anaconda/conda-bld/lomap_1491597173177/_t_env/lib/python2.7/site-packages/subprocess32.py:472: RuntimeWarning: The _posixsubprocess module is not being used. Child process reliability may suffer if your program uses threads.
  "program uses threads.", RuntimeWarning)
Traceback (most recent call last):
  File "/anaconda/conda-bld/lomap_1491597173177/test_tmp/run_test.py", line 27, in <module>
    import lomap
  File "/anaconda/conda-bld/lomap_1491597173177/_t_env/lib/python2.7/site-packages/lomap/__init__.py", line 62, in <module>
    from lomap.dbmol import DBMolecules
  File "/anaconda/conda-bld/lomap_1491597173177/_t_env/lib/python2.7/site-packages/lomap/dbmol.py", line 46, in <module>
    from lomap import graphgen
  File "/anaconda/conda-bld/lomap_1491597173177/_t_env/lib/python2.7/site-packages/lomap/graphgen.py", line 51, in <module>
    import matplotlib.pyplot as plt
  File "/anaconda/conda-bld/lomap_1491597173177/_t_env/lib/python2.7/site-packages/matplotlib/pyplot.py", line 115, in <module>
    _backend_mod, new_figure_manager, draw_if_interactive, _show = pylab_setup()
  File "/anaconda/conda-bld/lomap_1491597173177/_t_env/lib/python2.7/site-packages/matplotlib/backends/__init__.py", line 32, in pylab_setup
    globals(),locals(),[backend_name],0)
  File "/anaconda/conda-bld/lomap_1491597173177/_t_env/lib/python2.7/site-packages/matplotlib/backends/backend_qt5agg.py", line 16, in <module>
    from .backend_qt5 import QtCore
  File "/anaconda/conda-bld/lomap_1491597173177/_t_env/lib/python2.7/site-packages/matplotlib/backends/backend_qt5.py", line 26, in <module>
    import matplotlib.backends.qt_editor.figureoptions as figureoptions
  File "/anaconda/conda-bld/lomap_1491597173177/_t_env/lib/python2.7/site-packages/matplotlib/backends/qt_editor/figureoptions.py", line 20, in <module>
    import matplotlib.backends.qt_editor.formlayout as formlayout
  File "/anaconda/conda-bld/lomap_1491597173177/_t_env/lib/python2.7/site-packages/matplotlib/backends/qt_editor/formlayout.py", line 56, in <module>
    from matplotlib.backends.qt_compat import QtGui, QtWidgets, QtCore
  File "/anaconda/conda-bld/lomap_1491597173177/_t_env/lib/python2.7/site-packages/matplotlib/backends/qt_compat.py", line 137, in <module>
    from PyQt4 import QtCore, QtGui
ImportError: No module named PyQt4
TESTS FAILED: lomap-0.0.0-py27_1
```
Can we remove `lomap`, or can you get someone to fix this?

**Comment by davidlmobley** [comment]
Working on getting it looked at.

**Comment by davidlmobley** [comment]
@nividic says he's fixed the py27 version. We don't have py3 support yet but are working on it.

**Comment by davidlmobley** [comment]
Ah, right, closed by https://github.com/omnia-md/conda-recipes/pull/756 .

**Comment by davidlmobley** [comment]
(I don't appear to have permission to mark this as closed, but, it was resolved by #756 )
